custom_dc/troubleshooting.md

```diff
@@ -93,8 +86,18 @@ This error indicates that your application requires authenticated requests but y
 
 If you are unable to select this option, this indicates that there is an IAM permissions setup issue with your project or account. See the [Cloud Run Troubleshooting](https://cloud.google.com/run/docs/troubleshooting#unauthorized-client) for details on how to fix this.
 
+### "403: Not authorized to access resources"
+
```

Comment:
Should this be removed? Looks like it is covered under the next section.

Reply:
Yep.